tools/jenkins/include/scripts/build-Linux-sdk.sh

```diff
@@ -2945,117 +2949,117 @@ if [ ! -s "$SDK_HOME/bin/gdb" ]; then
-
-    if [ ! -d "qtbase" ]; then
-        git_clone_commit "$QTBASE_GIT" tags/Maya2018
-        pushd qtbase
-        git submodule update -i --recursive --remote
-        patch -p0 -i "$INC_PATH/patches/Qt/qtbase-revert-1b58d9acc493111390b31f0bffd6b2a76baca91b.diff"
-        patch -p0 -i "$INC_PATH/patches/Qt/qtbase-threadpool.diff"
-        popd
-    fi
-    pushd qtbase
-
-    QT_LD_FLAGS="-L${SDK_HOME}/lib"
-    if [ "$BITS" = "32" ]; then
-        QT_LD_FLAGS="$QT_LD_FLAGS -L${SDK_HOME}/gcc/lib"
-    else
-        QT_LD_FLAGS="$QT_LD_FLAGS -L${SDK_HOME}/gcc/lib64"
-    fi
-    env CFLAGS="$BF" CXXFLAGS="$BF" LDFLAGS="$QT_LD_FLAGS" ./configure -prefix "$QT5PREFIX" "${QT_CONF[@]}"
-    LD_LIBRARY_PATH="$(pwd)/lib${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}" make -j${MKJOBS}
-    make install
-    QTBASE_PRIV=( QtCore QtDBus QtGui QtNetwork QtOpenGL QtPlatformSupport QtPrintSupport QtSql QtTest QtWidgets QtXml )
-    for i in "${QTBASE_PRIV[@]}"; do
-        (cd "$QT5PREFIX/include/$i"; ln -sfv "$QT5_VERSION/$i/private" .)
-    done
-
-    popd
-    rm -rf qtbase
-    end_build "qt5-${QT5_VERSION}"
-
-    for (( i=0; i<${#QT_MODULES[@]}; i++ )); do
-        module=${QT_MODULES[i]}
-        moduleurl=${QT_MODULES_URL[i]}
-        start_build "${module}-${QT5_VERSION}"
-        package="${module}-${QT5_VERSION}.tar.gz"
-        if [ ! -s "$SRC_PATH/$package" ]; then
-            echo "*** downloading $package"
-            $WGET --no-check-certificate "$moduleurl" -O "$SRC_PATH/$package" || rm "$SRC_PATH/$package"
-        fi
-        if [ ! -s "$SRC_PATH/$package" ]; then
-            echo "*** Error: unable to download $package"
-            exit 1
-        fi
-        untar "$SRC_PATH/$package"
-        dirs=(${module}-*)
-        if [ ${#dirs[@]} -ne 1 ]; then
-            echo "Error: more than one match for ${module}-*: ${dirs[*]}"
-            exit 1
-        fi
-        dir=${dirs[0]}
-        pushd "$dir"
-        if [ "$module" = "qtdeclarative" ]; then
-            patch -p1 -i "$INC_PATH/patches/Qt/qtdeclarative-5.6.1-backport_core_profile_qtdeclarative_5.6.1.patch"
-        fi
-        if [ "$module" = "qtx11extras" ]; then
-            patch -p1 -i "$INC_PATH/patches/Qt/qtx11extras-5.6.1-isCompositingManagerRunning.patch"
-            patch -p1 -i "$INC_PATH/patches/Qt/qtx11extras-5.6.1-qtc-wip-add-peekeventqueue-api.patch"
-        fi
-        "$QT5PREFIX/bin/qmake"
-        make -j${MKJOBS}
-        make install
-        popd
-        rm -rf "$dir"
-
-        # Copy private headers to includes because Pyside2 needs them
-        ALL_MODULES_INCLUDE=("$QT5PREFIX"/include/*)
-        for m in "${ALL_MODULES_INCLUDE[@]}"; do
-            (cd "$m"; ln -sfv "$QT5_VERSION"/*/private .)
-        done
-
-        end_build "${module}-${QT5_VERSION}"
-    done
-fi
+# if [ "${REBUILD_QT5:-}" = "1" ]; then
```

Comment:
please enclose this portion within a `if false; then` ... `fi` block, without reindenting. This helps keeping track of file history